PracticeProject/PracticeProject/CardBoxView.swift

```diff
@@ -6,14 +6,20 @@
 //
 
 import SwiftUI
+import UIKit
 
 struct CardBoxView: View {
+    
     @State private var cardNumber: String = ""
     @State private var expiryDate: String = ""
     @State private var securityCode: String = ""
-    //@Binding private var image: Image?
-    //@State var isSelected: Bool = true
-    
+    @State var images: [Image] = [
```

Comment:
@SagarTilekar86 What are these images used for? Why is the first element a `UIKIt, UIImage` and what's the reason this property is `@State` property?

Reply:
Hi Lukas,
These are just system images used for testing.
Actually, I was trying for UIImage for that purpose I used UIKIt  I will remove it.
Kept the property as @State because I thought it was dynamic I will remove it.
Also Lukas can we have call?.

Thanks & Regard
Sagar Tilekar

Reply:
I think maybe after the meeting in half an hour